csfieldguide/static/interactives/password-guesser/js/passwords.js

```diff
@@ -31,16 +31,16 @@ var LanguagePasswords = [
-  ["Isabelle", "8C6976E5B5410415BDE908BD4DEE15DFB167A9C873FC4BB8A81F6F2AB448A918", "gdDAOlnmLtEeSlhY", "0E66D5C838959F2A00C55D392281401FB55452CDF11F802D767D054EFFC1AF55"], // admin
-  ["Jack",     "97C94EBE5D767A353B77F3C0CE2D429741F2E8C99473C3C150E2FAA3D14C9DA6", "rRiTDOYooATVXmdR", "439464EF67EA97952C588A2C4032756D7F2CC8CC7FC24619AEA0D85FD2A91FD4"], // Pa$$w0rd
-  ["Kate",     "B5C5FA751BFE75D18F266DBB7C11C4EC5A3A4E788AB64217821FB490642488B3", "piHNbJunE1U1zorO", "C23720935587E3CE2100A5B3867A60B3648F8A514167375084034CD37C28C01A"], // H0m3wOr/
-  ["Liam",     "0A66B00BB789BE6526CCB0BD3F8247CA3925CEFBC5BD43AD8968A0B0BE63861E", "uIKp9FjJ6QsF6qvS", "BA9FC99BAC73C7240D3F33D154E96F7943C963BE2F97DD0C50E433F374AC44BA"], // 0xC0FFEE
-  ["Michaela", "2727C279420E42CC2B7403DA760F7E62A97B1312E2BA34D88E28CCB6CF7B37AF", "kojuVmBIvkQCzrXp", "473D31001A251E7C8CBA52F02D4CF61FFE1253C5E2B48DDB2EAEA1DC074A5AD1"], // M!38@ela
-  ["Nat",      "9C97925A4F70A1C58EBA7A2D6E6ED3FE76FA5EB76AC2A2700E36429F4A5F8F80", "uLep9JTiRmM1T9gr", "93FF69837283764A87C8F442588ABAFF640C317EC10971E74EDA9E36ABF187D3"], // NAT!2x7A
+  ["Evelyn",   "5994471ABB01112AFCC18159F6CC74B4F511B99806DA59B3CAF5A9C173CACFC5", "4B9iTLSL27THrK0M", "593A4CA0674F8729B7192F48801E765B60430C61F07303D01505010D2AEADFA3"],
+  ["Frank",    "5994471ABB01112AFCC18159F6CC74B4F511B99806DA59B3CAF5A9C173CACFC5", "9pUOuSMy8g5gosp7", "22CB49E86CE07869D909D30C40566E15B10118F33F65F048092A1946F3843447"],
+  ["Grant",    "65E84BE33532FB784C48129675F9EFF3A682B27168C0EA744B2CF58EE02337C5", "B3W2w9A16ZTC0qKc", "EF6B7840D5219F7CE78E84039A1EE18F63173795BD16F31107E4B01E7DD49241"],
+  ["Hannah",   "04F8996DA763B7A969B1028EE3007569EAF3A635486DDAB211D512C85B9DF8FB", "F0LGkqngVY2i3WPP", "76FD57D1416244A0FBEA3718AD972420B4AF10E9DF73F71E10BB3E1FB1EB2540"],
+  ["Isabelle", "8C6976E5B5410415BDE908BD4DEE15DFB167A9C873FC4BB8A81F6F2AB448A918", "gdDAOlnmLtEeSlhY", "0E66D5C838959F2A00C55D392281401FB55452CDF11F802D767D054EFFC1AF55"],
+  ["Jack",     "97C94EBE5D767A353B77F3C0CE2D429741F2E8C99473C3C150E2FAA3D14C9DA6", "rRiTDOYooATVXmdR", "439464EF67EA97952C588A2C4032756D7F2CC8CC7FC24619AEA0D85FD2A91FD4"],
+  ["Kate",     "9184796BD1CF143C86029124DC0673097181F5C9F35A997B943F1A708E5D8C98", "piHNbJunE1U1zorO", "A6DF90D91F5EFE7DAC98F797135E3D872176D3416D25318BC7F1AF704466261A"],
+  ["Liam",     "0A66B00BB789BE6526CCB0BD3F8247CA3925CEFBC5BD43AD8968A0B0BE63861E", "uIKp9FjJ6QsF6qvS", "BA9FC99BAC73C7240D3F33D154E96F7943C963BE2F97DD0C50E433F374AC44BA"],
+  ["Michaela", "2727C279420E42CC2B7403DA760F7E62A97B1312E2BA34D88E28CCB6CF7B37AF", "kojuVmBIvkQCzrXp", "473D31001A251E7C8CBA52F02D4CF61FFE1253C5E2B48DDB2EAEA1DC074A5AD1"],
+  ["Nat",      "9C97925A4F70A1C58EBA7A2D6E6ED3FE76FA5EB76AC2A2700E36429F4A5F8F80", "uLep9JTiRmM1T9gr", "93FF69837283764A87C8F442588ABAFF640C317EC10971E74EDA9E36ABF187D3"],
```

Comment:
Nat's hash is calculated incorrectly

![image](https://user-images.githubusercontent.com/16066822/67720178-2822e980-fa38-11e9-9692-4f428113e4f2.png)

Reply:
It uses Michaela's salt. Will fix